test/modules/stats-widget/stats-widget.spec.js

```diff
@@ -16,14 +15,18 @@ const statsContent = [
     type: 'totalTime'
   },
   {
-    name: 'Memory',
+    name: 'GPU Memory',
     type: 'memory'
   }
 ];
 
-statsContent.forEach(({name, type}) => {
-  stats.get(name, type);
-});
+function getStatsObject() {
+  const stats = new Stats({id: 'test-stats'});
```

Comment:
Wouldn't it be nice to allow stats to be added in the constructor?

```js
const stats = new Stats({id: 'test-stats', stats: [
  {name 'GPU Memory', type: 'memory'},
  ...
]});
```

Reply:
Follow up PR